core/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -5865,6 +5874,9 @@ public static ConfigBuilder configBuilder() {
      * the relational expressions. Default is
      * {@link HintStrategyTable#EMPTY}. */
     HintStrategyTable getHintStrategyTable();
+
+    /** control keep sort in sub-query. */
+    boolean isKeepSortInSubQuery();
```

Comment:
```java
/**
 * Returns whether to keep sort operator for a sub-query if the sort has no offset and fetch limit attributes. 
 * Because the remove does not change the semantics, and in many cases, this is a promotion.
 * 
 * <p> Default is true
*/
```

I would suggest to rename it to `isRemoveSortInSubquery` if the default value for it is true.

Reply:
> ```java
> /**
>  * Returns whether to keep sort operator for a sub-query if the sort has no offset and fetch limit attributes. 
>  * Because the remove does not change the semantics, and in many cases, this is a promotion.
>  * 
>  * <p> Default is true
> */
> ```
> 
> I would suggest to rename it to `isRemoveSortInSubquery` if the default value for it is true.

Thanks for your advice, revised.